student/src/main/java/tds/student/web/handlers/TestShellHandler.java

```diff
@@ -231,31 +236,7 @@ private String parseOutLatencies (Map<String, String> formParams) {
 
     TestOpportunity testOpp = StudentContext.getTestOpportunity ();
 
-    // get responses
```

Comment:
Why are we removing all this additional logic on the student side? I was under the impression testManager.CheckIfTestComplete() (line 239) and testManager.LoadResponses() (line 236) were already implemented. TestManager.CheckIfTestComplete goes through nearly the same logic flow that the new /review endpoint does (Checking if segments are "satisfied")